public/i18n/de.json

```diff
@@ -191,6 +194,11 @@
         "LICENSE_LINK_AFTER": "gefunden werden"
     },
     "settings": {
-        "SETTINGS":"Einstellungen"
+        "SETTINGS":"Einstellungen",
+        "notifications": {
+            "NOTIFICATIONS" : "Benachrichtigungen",
+            "SHOW_NOTIFICATIONS" : "Desktopbenachrichtigungen anzeigen",
+            "SHOW_PREVIEW" : "Vorschau anzeigen"
```

Comment:
Maybe it would be clearer to the user what this does if it said "Nachrichteninhalt in Benachrichtigung anzeigen"? And in English, "Show message contents in notifications".